### DISCUSSION TITLE
Multi-task learning and ONNX support

### DISCUSSION THREAD

**Discussion by ogencoglu** [StatMixedML/XGBoostLSS#46] [Ideas]
Thanks for the great work!

I have two questions: 

1) Can we perform multi-task learning in a single training where one task is classification and the target variables are categorical (classes) and the other task is regression where the target values are continuous?

2) Does XGBoostLSS models support ONNX conversions?

**Comment by StatMixedML** [comment]
@ogencoglu Thanks for your interest in the project!

Concerning your questions

1. Interesting suggestion! Conceptually, this should be doable (not quite sure though). We need a common loss function though. Not sure if it is possible to add the NLL from the continuous part to the NLL from the categorical part. 
2. I haven't really tried, but feel free!
